bin/test-workflow/dev-start.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# This script is intended for use by developers and not to be called by automation.
+
+set -euo pipefail
+IFS=$'\n\t'
+export GCLOUD_PROJECT_NAME="${GCLOUD_PROJECT_NAME:-gke}"
+export GCLOUD_ZONE="${GCLOUD_ZONE:-gke}"
+export GCLOUD_CLUSTER_NAME="${GCLOUD_CLUSTER_NAME:-gke}"
+export GCLOUD_SERVICE_KEY="${GCLOUD_SERVICE_KEY:-gke}"
+
+print_usage() {
+  echo "Usage:"
+  echo "    This script will run start for the various platforms"
+  echo ""
+  echo "Syntax:"
+  echo "    $0 [Options]"
+  echo "    Options:"
+  echo "    -c                         Run in a container"
+  echo "    -e <Summon environment>    The environment used in"
+  echo "                               secrets.yml. Defaults to dev"
+  echo "    -g                         GKE"
+  echo "    -h                         Show help"
+  echo "    -l                         Just Log in to the platform"
+  echo "    -o                         Openshift"
+  echo "    -n                         No cleanup"
+  echo "    -s <Step #>                Run one step"
```

Comment:
Your call, but maybe "Run one of the numbered scripts" for someone new to these workflow scripts?